.github/dependabot.yml

```diff
@@ -13,14 +13,7 @@ updates:
     labels:
       - dependencies
     groups:
-      eslint:
+      biome:
         dependency-type: "development"
         patterns:
-          - "eslint"
-          - "@eslint/compat"
-          - "@eslint/eslintrc"
-          - "@eslint/js"
-          - "eslint-config-prettier"
-          - "eslint-import-resolver-typescript"
-          - "eslint-plugin-*"
-          - "@typescript-eslint/*"
+          - "@biomejs/biome"
```

Comment:
You can drop the group, no added value to have a group with a single dependency

Reply:
I removed the group, never used dependabot so not sure, if thats what you had in mind.